wagtail_transfer/vendor/wagtail_api_v2/serializers.py

```diff
@@ -329,6 +329,27 @@ def build_relational_field(self, field_name, relation_info):
         return super().build_relational_field(field_name, relation_info)
 
 
+class GenericModelSerializer(serializers.ModelSerializer):
+    """Generic Model Serializer. Pass in the keyword `model` to activate it."""
+
+    object_name = serializers.CharField(source='__str__')
```

Comment:
I avoided using `name` because that's a common field name and we might not want to overwrite that so I went with a verbose name that is unlikely to be used as a Django field. But the `label` -> `model_label` clearly doesn't follow that same logic so Ill rename the `model_label`.